src/plugins/GUITestBase/src/GTUtilsMsaEditor.h

```diff
@@ -180,6 +180,10 @@ class GTUtilsMsaEditor {
 
     /** Returns active Exclude List widget instance. */
     static QListWidget* getExcludeListWidget(HI::GUITestOpStatus& os);
+
+    // Press "Multiline View" button on toolbar
+    static bool getMultilineMode(HI::GUITestOpStatus &os);
+    static void setMultilineMode(HI::GUITestOpStatus &os, bool mode);
```

Comment:
Add comment to all public methods. I can't see from the method declaration here what happens when I call setMultilineMode & multiline mode is already ON.

Reply:
done